src/Concerns/ManagesCustomer.php

```diff
@@ -64,4 +67,44 @@ public function lemonSqueezyTaxNumber(): ?string
     {
         return $this->tax_number ?? null; // 'GB123456789'
     }
+
+    /**
+     * Get the customer portal url for this billable.
+     */
+    public function customerPortalUrl(): string
+    {
+        $this->assertCustomerExists();
+
+        $response = LemonSqueezy::api('GET', "customers/{$this->lemon_squeezy_id}");
```

Comment:
Shouldn't this be

`$response = LemonSqueezy::api('GET', "customers/{$this->customer->lemon_squeezy_id}");`

Reply:
@dylanmichaelryan you're totally right. I've released a fix for this.